data_visualization_demo.ipynb

```diff
@@ -0,0 +1,3079 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "Blue.plot.pcolormesh('Easting', 'Northing', robust=True, add_colorbar=False)\n",
+    "plt.title('Blue band');"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 73,
+   "id": "9a597caa-bdc7-48df-844a-750e346fa254",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Select an RGB composite \n",
+    "RGB = ds.sel(wavelength=[665,550,440], method='nearest').Reflectance"
```

Comment:
Same here for 670. We can also divide RGB by 0.7 to lighten up the image to see better.